proto/cluster.proto

```diff
@@ -216,88 +219,89 @@ message Volume {
   map<string, string> items = 11; // Items used for configMap and secrets
 }
 
+// Cluster HeadGroup specification
 message HeadGroupSpec {
-  // Optional. The computeTemplate of head node group
-  string compute_template = 1;
+  // Required. The computeTemplate of head node group
+  string compute_template = 1 [(google.api.field_behavior) = REQUIRED];
   // Optional field. This field will be used to retrieve right ray container
-  string image = 2;
+  string image = 2 [(google.api.field_behavior) = OPTIONAL];
   // Optional. The service type (ClusterIP, NodePort, Load balancer) of the head node
-  string service_type = 3;
+  string service_type = 3 [(google.api.field_behavior) = OPTIONAL];
   // Optional. Enable Ingress
   // if Ingress is enabled, we might have to specify annotation IngressClassAnnotationKey, for the cluster itself, defining Ingress class
-  bool enableIngress = 4;
+  bool enableIngress = 4 [(google.api.field_behavior) = OPTIONAL];
   // Optional. The ray start params of head node group. 
-  map<string, string> ray_start_params = 5;
+  map<string, string> ray_start_params = 5 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
Should HeadGroupSpec.start_params be required? https://github.com/z103cb/kuberay/blob/23303f4902cc2a37da28653ff637b32fe81c511f/helm-chart/kuberay-operator/crds/ray.io_rayclusters.yaml#L3529-L3530

Reply:
Good catch, will fix.

Reply:
Fixed.